docs/storage/multipathing.md

```diff
@@ -0,0 +1,359 @@
+---
+sidebar_position: 1
+---
+
+# Multipathing
+
+How to properly setup a new SR with multipathing on Xen Orchestra and XCP-ng.
+
+:::warning
+Do not attempt to enable multipathing on a production pool with existing and active iSCSI and/or HBA and/or FC SRs.
+:::
+
+## iSCSI
+
+### Requirements
+* Two different network interfaces
+* Two different switches
+* Multiple targets per LUN on your storage unit
+* iSCSI target ports are operating in portal mode
+
+:::info
+If the storage vendor recommends using Jumbo Frames, you will need to implement them.
+
+Since each architecture is unique, feel free to check with the storage vendor if it’s possible to stay with an MTU of 1500 (e.g., using storage dedicated PIF at 10Gb/s or 25Gb/s).
```

Comment:
```suggestion
Since each architecture is unique, feel free to check with the storage vendor if it’s possible to stay with the default 1500 MTU (e.g., using storage dedicated PIF at 10Gb/s or 25Gb/s), as this enables a simpler setup, less prone to misconfigurations, and because with modern hardware MTU has less impact than it used to.
```